.evergreen/resync-specs.sh

```diff
@@ -0,0 +1,145 @@
+#!/bin/bash
+# exit when any command fails
+set -e
+PYMONGO=$(dirname $PWD)
```

Comment:
PWD is the current working directory which isn't necessarily the repo root. Instead we should use the path to this file itself:
```
PYMONGO=$(dirname $(dirname $0))
```

Reply:
Sorry this is what works:
```
PYMONGO=$(dirname $(cd $(dirname $0); pwd))
```